go.mod

```diff
@@ -105,6 +105,7 @@ require (
 	github.com/keeper-security/secrets-manager-go/core v1.6.4
 	github.com/lestrrat-go/jwx/v2 v2.1.6
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.11.2
+	github.com/ngrok/ngrok-api-go/v7 v7.4.0
```

Comment:
nit: @gusfcarvalho I think we really need to push out-of-tree provider engineering. Maintaining this long-term along all other dependencies can cause trouble